contracts/gov/GovCreator.sol

```diff
@@ -168,6 +174,27 @@ abstract contract GovCreator is IGovCreator {
         }
     }
 
+    function _handleDataForChangeValidatorBalanceProposal(
+        address[] calldata executors,
+        uint256[] calldata values,
+        bytes[] calldata data
+    ) private view {
+        require(
+            address(govValidators) == executors[executors.length - 1],
+            "GovC: invalid executor"
+        );
```

Comment:
This require is useless, this function won't be called if the executor is not a validator